bliss/models/decoder.py

```diff
@@ -580,7 +503,15 @@ def _render_single_galaxies(self, galaxy_params, galaxy_bools):
 
         # forward only galaxies that are on!
         # no background
-        gal_on = self.galaxy_decoder(z[b == 1])
+        if lens_params is not None:
```

Comment:
another `if` statement on `lens_params`, we should talk about whether we can remove this...